pkg/orchestrator/config/config.go

```diff
@@ -200,13 +197,14 @@ func setBoundedConfigIntValue(configKey string, upperBound int, setter func(v in
 }
 
 // IsOrchestratorEnabled checks if orchestrator explorer features are enabled, it returns the boolean and the cluster name
```

Comment:
the description should be updated as it returns a new bool

Reply:
[16a0135](https://github.com/DataDog/datadog-agent/pull/18740/commits/16a013528fde6d4f2ec188c256d6f9eb26d033dc)